web/styles/zulip.css

```diff
@@ -226,6 +226,7 @@ body {
     --color-background-tab-picker-selected-tab: hsl(0deg 0% 100%);
     --color-outline-tab-picker-tab-option: hsl(0deg 0% 0% / 30%);
     --color-background-tab-picker-tab-option-hover: hsl(0deg 0% 100% / 60%);
+    --color-background-popovoer-light-theme: hsl(0deg 0% 0%);
```

Comment:
Not used right now, but will be used when we migrate it to use colors in navbar dropdown.